drivers/hud_spm_report/app/models/hud_spm_report/filters/spm_filter.rb

```diff
@@ -5,22 +5,6 @@
 ###
 
 module HudSpmReport::Filters
-  class SpmFilter < ::Filters::FilterBase
-    validates_presence_of :coc_codes
-
-    # NOTE: This differs from the base filter class because it includes all projects based on project type, and doesn't include any projects based on CoCCode
-    def effective_project_ids
-      @effective_project_ids ||= begin
-        ids = effective_project_ids_from_projects
-        ids += effective_project_ids_from_project_groups
-        ids += effective_project_ids_from_organizations
-        ids += effective_project_ids_from_data_sources
-        ids = all_project_ids if ids.empty?
-
-        ids = ids.uniq.reject(&:blank?)
-        ids &= effective_project_ids_from_project_types if effective_project_ids_from_project_types.present?
-        ids
-      end
-    end
+  class SpmFilter < ::Filters::HudFilterBase
```

Comment:
Sorry the request on the CeAprFilter was incomplete, any of these filters that just inherit and don't add any benefit should be replaced.